workers/data_refinery_workers/processors/illumina.yml

```diff
@@ -0,0 +1,32 @@
+---  # "Illumina SCAN" processor
```

Comment:
Are you saying to keep track of the commit hash of `processors/illumina.R`?

Reply:
Yes

Reply:
In my mind, this is a unique case because it is an implementation of what is in the `SCAN.UPC` package but for a different platform and it is not yet included in the package itself. It's the only spot that I can think of where a script is doing the heavy lifting of data processing rather than just calling functions from a package. That being said, I'm not dead set on this if it turns out to be too much of a hassle.

Reply:
I wrote a function to get its md5sum for this file. Stay tuned for the updated PR. (May not be able to finish it by today due to some unexpected complexities though.)